src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1124,6 +1127,16 @@ private IExpressionStatement CreateBoundExpressionStatementOperation(BoundExpres
             return new LazyExpressionStatement(expression, isInvalid, syntax, type, constantValue);
         }
 
+        private ITupleExpression CreateBoundTupleExpressionOperation(BoundTupleExpression boundTupleExpression)
+        {
+            Lazy<ImmutableArray<IOperation>> elements = new Lazy<ImmutableArray<IOperation>>(() => boundTupleExpression.Arguments.SelectAsArray(element => Create(element)));
+            bool isInvalid = boundTupleExpression.HasErrors;
+            SyntaxNode syntax = boundTupleExpression.Syntax;
+            ITypeSymbol type = boundTupleExpression.Type;
+            Optional<object> constantValue = ConvertToOptional(boundTupleExpression.ConstantValue);
```

Comment:
Re-opening question from previous PR: tuples don't have constant values.
@cyrusnajmabadi prefered to keep as-is. But I don't see the benefit.